.github/workflows/cicd-main.yml

```diff
@@ -2895,7 +2895,7 @@ jobs:
     with:
       RUNNER: self-hosted-azure-gpus-2-h100
       SCRIPT: |
-        CUDA_DEVICE_MAX_CONNECTIONS=1 NVTE_FLASH_ATTN=0 NVTE_FUSED_ATTN=1 python examples/nlp/language_modeling/tuning/megatron_gpt_finetuning.py \
```

Comment:
Looks good, only catch is , if fused attention didnt work for this setting, TE would have automatically used unfused attention. So maybe if you still see failure, it might be that TE used unfused attention.